README.md

```diff
@@ -189,6 +189,18 @@ For instance:
 If you use `target.merkle.tree.file.name` it will render the string
 `foobar` and `target.proof.hash` will render `abcd`.
 
+### Assets path url
+
+By default this package assumes the static admin will be hosted at the root of a given domain; if you plan on hosting it under a given sub path, you need to build the admin setting the `KINTO_ADMIN_PUBLIC_PATH` env var, specifying the absolute root URL path where the static asset files are obtainable.
+
+For example, if you plan on hosting the admin at `https://mydomain.tld/kinto-admin/`, you need to build it like this:
+
+```
+$ KINTO_ADMIN_PUBLIC_PATH=/kinto-admin/ npm run dist
+```
+
+Note: Unfortunately, the `kinto-admin build` CLI command doesn't support this feature.
```

Comment:
Not easily, that's why for this purpose right now the solution is to build the admin itself setting the `KINTO_ADMIN_PUBLIC_PATH` var. Note that this is mostly matter of working from the repo instead of the installed package, running:

```
$ KINTO_ADMIN_PUBLIC_PATH=/plop/ npm run dist 
```

Instead of

```
$ kinto-admin build
```

I understand and agree that this is a little less convenient, though moving the package executable to on-demand building is a big and schedule-risky task really.

Note: Depending on the future of plugins, we could also consider this opportunity to offer on-demand building of the admin with selected plugins as well, so that would be killing two (large) birds with one stone. I wasn't expecting to work on it this soon, but that may be the right moment to start the move. 

Now the embarrassing question: would you want to work on this? :)